config/volume.go

```diff
@@ -405,3 +405,17 @@ func (cfg *Volume) ToDriverOptions(timeout time.Duration) (storage.DriverOptions
 func (cfg *Volume) String() string {
 	return path.Join(cfg.PolicyName, cfg.VolumeName)
 }
+
+// IsVolumeInUse checks if the given volume is mounted in any container
+func (c *Client) IsVolumeInUse(cfg *Volume) (bool, error) {
+	if !cfg.Unlocked {
+		if _, err := c.etcdClient.Get(context.Background(), c.use(UseTypeMount, cfg.String()), nil); err != nil {
+			if cErr, ok := err.(client.Error); ok {
+				return !(cErr.Code == client.ErrorCodeKeyNotFound), nil
+			}
+			return true, err
+		}
+	}
+	// XXX To simplify the behavior, always return true for unlocked volumes as there are no mount lock exists for them.
```

Comment:
If you check if the volume is unlocked and return at the beginning, you can get rid of one level of indentation in most of the function.  Not sure what is stylistically best here.